octopusdeploy/schema_deployment_step.go

```diff
@@ -1,6 +1,8 @@
 package octopusdeploy
 
 import (
+	"log"
```

Comment:
I wonder if `github.com/hashicorp/terraform-plugin-log/tflog` is a better package to use

Reply:
I did use that at first but it required passing around the context object around. From what I have read there isnt really much difference between them. At the very least we seem to use both right now so im not sure which is "the best" TBH